src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -1500,4 +1504,54 @@ class TcgStatics {
     }
   }
 
+  /**
+   * Scoop up PokemonCardSets if not blocked by Scoop-Up Block
+   *
+   * @param params Optional settings map
+   *  + discard: Indicates if any cards should be discarded. Default: true
+   *  + only: A card or list of cards to move to hand instead of discard.
```

Comment:
This parameter design looks complicated to me. 

It'd look simpler like this;

- all: boolean - if true, scoops up all cards, else all pokemon cards. discards the rest.
- only: cardlist - only scoops up them, discards the rest.

Reply:
Like this? https://github.com/axpendix/tcgone-engine-contrib/pull/921/commits/c8e17c35fd8d48e16b9975d37abc46a754294edb

Reply:
Don't you think 'all' should be false by default? It would follow default variable conventions.

Reply:
I have conflicting thoughts on this. You're right, it would. It was something that was bothering me when I was calling it discard instead. Particularly because the majority of Pokémon scoop up effects (including ones in the main repo) seem to scoop up all of the cards and scooping up only the Pokémon seems to be a rarer case. I feel that making the most common case the default, and using the params for progressively different behaviour makes it easier to implement any future effects IMO.

If you feel that the default variable convention is more important than that though, I can make that change.

Reply:
Agree with the common case. Then the parameter should better be 'pokemonOnly' ?